### ISSUE TITLE
Default value is not applied in the rendered form

### ISSUE THREAD

**Issue by dkalogirou** [merobal/angular-formio-custom-demo#1]
Hi merobal, 

I tried to apply a default value in the rating component from the form builder of your example and the value is not applied in the renderer. Is this some bug or is the expected behaviour of the angular formio component. Can we set the default value in builder?

Thank you!

**Comment by dmenne** [comment]
Try to set "Redraw On" to "AnyChange" on the data tab of the component. I do not know why it is needed, things work without in angular-form-custom-demo. I have check dozens of time what is different in my, no way.  Any idea?

**Comment by merobal** [comment]
Do you use Angular Elements 8.1.3 or newer?
Can you please provide a repro example?

**Comment by dmenne** [comment]
Thanks for your work to get custom components off ground. I find it disappointing that the formio team does not care; I see, it does not make money, but custom components could spin up a cottage industry.

I am currently a bit lost in the "recursive browser hell" reported in a separate thread - I had not realized it is a core formio problem.

So a bit half-guessing: Remove this, it is not realistic in a full scenario

  <div *ngIf="!builder">
      <formio [form]="form"
              (change)="onChange($event)"></formio>
    </div>

Usual components still render, but rating does not. When you set Redraw On to "AnyChange", it will turn up. So we are looking how to post that .

**Comment by merobal** [comment]
Until they fix that issue, try to pin `angular-formio` version to `4.3.3` which uses `formiojs: 4.7.8` in the background which was a stable version.

(We are also stuck using that version so I'm not familiar with newer releases. Same infinite loop issue...)

**Comment by dmenne** [comment]
You are right. It also fixed my "own radio buttons for the clumsy tablet user" 
using https://ng-bootstrap.github.io/#/components/buttons/examples
